From rain_maker@root-forum.org  Fri Sep 11 15:22:15 2009
From: "Axel K." <rain_maker@root-forum.org>
Date: Thu, 3 Sep 2009 20:47:11 +0200
Subject: Staging: rt3090: add device id 1462:891a
To: Bartlomiej Zolnierkiewicz <bzolnier@gmail.com>
Cc: Greg KH <gregkh@suse.de>
Message-ID: <200909032047.12570.rain_maker@root-forum.org>
Content-Disposition: inline


This patch adds a new device ID (1462:819a) to ralink rt3090 driver in linux
staging. The device ID was retrieved from the latest vendor release (version
2.2.0.0).

Signed-off-by: Axel Koellhofer <rain_maker@root-forum.org>
Acked-by: Bartlomiej Zolnierkiewicz <bzolnier@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/rt3090/pci_main_dev.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/staging/rt3090/pci_main_dev.c
+++ b/drivers/staging/rt3090/pci_main_dev.c
@@ -83,6 +83,7 @@ static struct pci_device_id rt2860_pci_t
 	{PCI_DEVICE(NIC_PCI_VENDOR_ID, NIC3090_PCIe_DEVICE_ID)},
 	{PCI_DEVICE(NIC_PCI_VENDOR_ID, NIC3091_PCIe_DEVICE_ID)},
 	{PCI_DEVICE(NIC_PCI_VENDOR_ID, NIC3092_PCIe_DEVICE_ID)},
+	{PCI_DEVICE(0x1462, 0x891A)},
 #endif // RT3090 //
 #ifdef RT3390
 	{PCI_DEVICE(NIC_PCI_VENDOR_ID, NIC3390_PCIe_DEVICE_ID)},
